etna/models/sarimax.py

```diff
@@ -374,9 +381,9 @@ class _SARIMAXAdapter(_SARIMAXBaseAdapter):
 
     def __init__(
         self,
-        order: Tuple[int, int, int] = (2, 1, 0),
-        seasonal_order: Tuple[int, int, int, int] = (1, 1, 0, 12),
-        trend: Optional[str] = "c",
+        order: Tuple[int, int, int] = (1, 0, 0),
```

Comment:
Why did you change the default values?

Reply:
Because our values in a library doesn't correspond to the values in statsmodels library. There were two problems:
1. Our documentations contradicts the default values we set. For example, we write in documentation that by default we use AR(1) model, but we use ARIMA(2, 1, 0) model.
2. Our default values are rather strange. For what reason do we set `seasonal_order=(x, x, x, 12)`? Why do we set seasonality of 12?

Reply:
No ideas))